src/perl/Bio/EnsEMBL/Pipeline/Runnable/EG/LoadGFF3/AnalysisSetup.pm

```diff
@@ -179,7 +179,7 @@ sub production_updates {
   ));
   $sth->fetch();
   
-  $properties{'web_data'} = eval ($properties{'web_data'}) if defined $properties{'web_data'};
+  $properties{'web_data'} = eval ({$properties{'web_data'}}) if defined $properties{'web_data'};
```

Comment:
```suggestion
  $properties{'web_data'} = eval "$properties{'web_data'}" if defined $properties{'web_data'};
```
This is a string eval, not a block (we're trying to load a data structure from a string here)

Reply:
```suggestion
  $properties{'web_data'} = eval {print $properties{'web_data'}} if defined $properties{'web_data'};
```
Rather this